packages/sra-report/src/index.ts

```diff
@@ -0,0 +1,102 @@
+
+// extract command line arguments
+const args = yargs
+    .option('endpoint-url', {
+        alias: ['e'],
+        describe: 'API endpoint url to test for standard relayer API compliance',
+        type: 'string',
+        demandOption: true,
+    })
+    .option('output', {
+        alias: ['o', 'out'],
+        describe: 'The relative path to write the report generated by the collection run, prints to console by default',
+        type: 'string',
+        normalize: true,
+        demandOption: false,
+    })
+    .option('network-id', {
+        alias: ['n'],
+        describe: 'ID of the network that the API is serving orders from',
+        type: 'number',
+        default: DEFAULT_NETWORK_ID,
+    })
+    .option('environment', {
+        alias: ['env'],
+        describe: 'The relative path to a postman environment file for the collection run',
+        type: 'string',
+        normalize: true,
+        demandOption: false,
+    })
+    .option('export-collection', {
+        alias: ['ec'],
+        describe: 'The relative path to write the postman collection file used by the collection run',
+        type: 'string',
+        normalize: true,
+        demandOption: false,
+    })
+    .option('export-environment', {
+        alias: ['ee'],
+        describe: 'The relative path to write the postman environment file used by the collection run',
+        type: 'string',
+        normalize: true,
+        demandOption: false,
+    })
+    .example(
+        "$0 --endpoint-url 'http://api.example.com' --out 'path/to/report.json' --network-id 42 --environment 'path/to/custom/environment.json' --export-collection 'path/to/collection.json' --export-environment 'path/to/environment.json'",
+        'Full usage example',
+    ).argv;
+// perform extra validation on command line arguments
+try {
+    assert.isWebUri('args', args.endpointUrl);
+} catch (err) {
+    utils.log(`${chalk.red(`Invalid url format:`)} ${args.endpointUrl}`);
+    process.exit(1);
+}
+if (!_.includes(SUPPORTED_NETWORK_IDS, args.networkId)) {
+    utils.log(`${chalk.red(`Unsupported network id:`)} ${args.networkId}`);
+    utils.log(`${chalk.bold(`Supported network ids:`)} ${SUPPORTED_NETWORK_IDS}`);
+    process.exit(1);
+}
+const mainAsync = async () => {
+    const newmanReporterOptions = !_.isUndefined(args.output)
+        ? {
+              reporters: 'json',
+              reporter: {
+                  json: {
+                      export: args.output,
+                  },
+              },
+          }
+        : {
+              reporters: 'cli',
+          };
+    const environment = !_.isUndefined(args.environment)
+        ? args.environment
+        : await postmanEnvironmentFactory.createPostmanEnvironmentAsync(args.endpointUrl, args.networkId);
+    const newmanRunOptions = {
+        collection: sraReportCollectionJSON,
+        environment,
+        exportCollection: args.exportCollection,
+        exportEnvironment: args.exportEnvironment,
```

Comment:
This seems to be exporting without the `name` attribute, causing imports into postman to fail.